runtime/lua/vim/_editor.lua

```diff
@@ -68,6 +68,13 @@ vim.log = {
   },
 }
 
+--- @private
+vim.utfs = {
```

Comment:
does this need to be on the public interface?

```suggestion
local utfs = {
```

Reply:
no, have made it local